draft-ietf-httpbis-http2bis.xml

```diff
@@ -1558,18 +1577,24 @@
           The PRIORITY frame (type=0x2) is deprecated; see <xref target="PriorityHere"/>.  A
           PRIORITY frame can be sent in any stream state, including idle or closed streams.
         </t>
-        <figure anchor="PRIORITYFramePayload">
-          <name>PRIORITY Frame Payload</name>
+        <figure anchor="PRIORITYFrameFormat">
+          <name>PRIORITY Frame Format</name>
           <artwork type="inline"><![CDATA[
- +-+-------------------------------------------------------------+
- |E|                  Stream Dependency (31)                     |
- +-+-------------+-----------------------------------------------+
- |   Weight (8)  |
- +-+-------------+
+Priority Frame {
+  Length (24),
+  Type (8),
+  Unused Flags (8),
+  R (1),
+  Stream Identifier (31),
+  E (1),
```

Comment:
And here.

Reply:
Yes please, this allows for "proper" names there!